Cargo.toml

```diff
@@ -4,5 +4,6 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+clap = { version = "4.4.11", features = ["derive"] }
```

Comment:
This maybe comes down to preference so I'll leave it to you - I typically only specify the major version in this case so I can pick up minor and patch releases. It does open yourself up to semver-violating breaking changes though.

Reply:
Yeah, that's better, I will change it. For some reason I thought you were using minor versions too :facepalm: 

But I would leave this for another PR since the drm/gbm deps are also outdated now and I'd like to upgrade them.

Reply:
There were no breaking changes, so I included this in this PR.